### ISSUE TITLE
Core.cs, WebException from RunRequest hides Response

### ISSUE THREAD

**Issue by nickcoad** [mozts2005/ZendeskApi_v2#247]
Just noticed that in Core.cs, if a WebException is detected in RunRequest it gets wrapped in a new WebException with some extra information in the message. This is very useful however the new WebException is missing the "Response" property so handling the exception is a little awkward. For example, to handle a 404 the following is required:

```
try
{
    var ticketResponse = _zendeskApi.Tickets.GetTicket(ticketNumber);
    if (ticketResponse == null || ticketResponse.Ticket == null)
        return false;

    return true;
}
catch (WebException we)
{
    var innerException = we.InnerException as WebException;
    if (innerException != null)
    {
        var httpWebResponse = innerException.Response as HttpWebResponse;
        if (httpWebResponse != null && httpWebResponse.StatusCode == HttpStatusCode.NotFound)
            return false;
    }
    throw;
}
```

I would recommend modifying the "GetWebException" method by adding on line 569:
```
wException.Response = originalWebException.Response;
```

**Comment by mozts2005** [comment]
Good catch if you would like to submit a pull request for this I would happily except it.

**Comment by mozts2005** [comment]
This issue is fixed #248
